server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -770,6 +770,25 @@ private List<File> buildPods() throws IOException, InterruptedException, Extende
             }
         }
 
+        LOGGER.info("buildPods - adding framework resource to build output");
+        File resourcesBuildDir = new File(buildDirectory, "resources");
+        resourcesBuildDir.mkdir();
+        List<String> resources = resolvedPods.getAllPodResources(platform);
+        for (String resource : resources) {
+            File resourceFile = new File(resource);
+            if (resourceFile.isFile()) {
```

Comment:
Resources can according to the pod spec be either individual resources or .bundle packages (ie folders)